cobigen-templates/templates-oasp4j/src/main/java/com/devonfw/cobigen/templates/oasp4j/utils/OaspUtil.java

```diff
@@ -1,18 +1,19 @@
-
 package com.devonfw.cobigen.templates.oasp4j.utils;
 
+import java.util.ArrayList;
 import java.util.Collection;
 import java.util.HashMap;
 import java.util.Map;
 import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import org.apache.xerces.dom.DeferredElementNSImpl;
```

Comment:
Please change it to `import com.sun.org.apache.xerces.internal.dom.DeferredElementNSImpl`